test/depayloader_pipeline_test.exs

```diff
@@ -10,31 +10,32 @@ defmodule Membrane.RTP.H264.DepayloaderPipelineTest do
 
   describe "Depayloader in a pipeline" do
     test "does not crash when parsing staps" do
-      {:ok, pid} =
+      {:ok, _supervisor_pid, pid} =
         STAPFactory.sample_data()
         |> Enum.chunk_every(2)
         |> Enum.map(&STAPFactory.into_stap_unit/1)
         |> Enum.map(&%Membrane.Buffer{payload: &1})
         |> Source.output_from_buffers()
         |> DepayloaderTestingPipeline.start_pipeline()
 
-      Membrane.Pipeline.play(pid)
+      # Membrane.Pipeline.play(pid)
+      Membrane.Testing.Pipeline.execute_actions(pid, playback: :playing)
 
       STAPFactory.sample_data()
       |> Enum.each(fn elem ->
         assert_sink_buffer(pid, :sink, buffer)
         assert %Buffer{payload: payload} = buffer
-        assert <<1::32, elem::binary()>> == payload
+        assert <<1::32, elem::binary>> == payload
       end)
 
-      Membrane.Pipeline.stop_and_terminate(pid, blocking?: true)
+      Membrane.Pipeline.terminate(pid, blocking?: true)
```

Comment:
Is this needed? You start pipeline using `start_link_supervised`

Reply:
I think it's required. `start_link` doesn't act when the process terminates cleanly.